pkg/cnab/provider/uninstall.go

```diff
@@ -42,12 +41,13 @@ func (d *Duffle) Uninstall(args ActionArguments) error {
 		}
 	}
 
-	driver, err := d.newDriver(args.Driver, c.Name)
+	driver, err := d.newDriver(args.Driver, c.Name, args)
 	if err != nil {
 		return errors.Wrap(err, "unable to instantiate driver")
 	}
 	i := action.Uninstall{
-		Driver: driver,
+		Driver:          driver,
+		OperationConfig: args.ApplyFiles(),
```

Comment:
This change here is part of a PR to cnab-go (see https://github.com/deislabs/cnab-go/pull/112). In the meantime, we'll use my fork.